doc/api.md

```diff
@@ -651,6 +651,16 @@ Example:
 console.log(mcData.legacy.pc.blocks['171:15']) // Returns: 'minecraft:black_carpet'
 ```
 
+### minecraftData.legacy.bedrock.blocks
+
+Mapping from 1.2 block:metadata to 1.18.10 block names
```

Comment:
This should explicitly mention 1.18.10 as 1.18 is ambiguous, given some blocks were changed in 1.18.* versions. I think it'd be better to map to the version immediately after flattening, but it's not a big deal right now. Other mapping data would need to be added also for other block renames/changes, but this can be done later to keep things simple for now.